modules/gltf/gltf_document.cpp

```diff
@@ -6224,7 +6224,9 @@ void GLTFDocument::_import_animation(Ref<GLTFState> p_state, AnimationPlayer *p_
 				if (p_remove_immutable_tracks) {
 					Vector3 base_pos = p_state->nodes[track_i.key]->position;
 					for (int i = 0; i < track.position_track.times.size(); i++) {
-						Vector3 value = track.position_track.values[track.position_track.interpolation == GLTFAnimation::INTERP_CUBIC_SPLINE ? (1 + i * 3) : i];
+						int value_index = track.position_track.interpolation == GLTFAnimation::INTERP_CUBIC_SPLINE ? (1 + i * 3) : i;
```

Comment:
```suggestion
						int value_index = (track.position_track.interpolation == GLTFAnimation::INTERP_CUBIC_SPLINE) ? (1 + i * 3) : i;
```
For readability, same below

Reply:
My bad, was preparing a batch of approved PRs and I missed this comment.